airflow/models/baseoperator.py

```diff
@@ -1239,6 +1239,36 @@ def chain(*tasks: Union[BaseOperator, List[BaseOperator]]):
             up_t.set_downstream(down_t)
 
 
+def chain_as_binary_tree(*tasks: BaseOperator):
```

Comment:
Can you move it to elastic_dag.py? We don't want other users to use this method because it is only useful for testing.

Reply:
fixed 👍